src/motions/mod.rs

```diff
@@ -143,7 +146,10 @@ pub fn create(directory: String, mut motions: Vec<Motion>, name: String) {
   println!("Writting: {:?} to the file system", path);
   if let Ok(mut f) = File::create(path.as_path()) {
     if let Err(e) = f.write_all(cookie.add.as_bytes()) {
-      panic!("Could not write {:?} to the file system due to:\n{}", path, e);
+      // TODO: Make this an `Err`.
+      panic!("Could not write {:?} to the file system due to:\n{}",
```

Comment:
Oops, I used `rustfmt` once with a short line length, I fixed most things, this one slipped through.